tests/Oryx.BuildImage.Tests/Node/NodeJsSampleAppsOtherTests.cs

```diff
@@ -23,9 +23,21 @@ public NodeJsSampleAppsOtherTests(ITestOutputHelper output) : base(output)
         {
         }
 
+        public static TheoryData<string> ImageNameData
+        {
+            get
+            {
+                var data = new TheoryData<string>();
+                data.Add(Settings.BuildImageName);
+                data.Add(Settings.SlimBuildImageName);
+                var imageTestHelper = new ImageTestHelper();
+                data.Add(imageTestHelper.GetAzureFunctionsJamStackBuildImage());
+                return data;
+            }
+        }
```

Comment:
can you add this image to that test where we check the commit and build information?